app/controllers/observations_controller.rb

```diff
@@ -99,41 +99,11 @@ def index
     # looked up now as it will use Angular/Node. This is for legacy
     # API methods, and HTML/views and partials
     if human_or_scraper && !showing_partial
-      search_taxon = Taxon.find_by_id( params[:taxon_id] ) unless params[:taxon_id].blank?
-      search_place = unless params[:place_id].blank?
-        Place.find( params[:place_id] ) rescue nil
-      end
-      search_user = unless params[:user_id].blank?
-        User.find_by_id( params[:user_id] ) || User.find_by_login( params[:user_id] )
-      end
-      search_date = Date.parse( params[:on] ) unless params[:on].blank?
-      @shareable_description = if search_taxon
-        key = "observation_brief_taxon"
-        i18n_vars = {}
-        i18n_vars[:taxon] = search_taxon.common_name( locale: I18n.locale ).try(:name)
-        i18n_vars[:taxon] = search_taxon.name if i18n_vars[:taxon].blank?
-        if search_place
-          key += "_from_place"
-          i18n_vars[:place] = search_place.localized_name
-        end
-        if search_date
-          key += "_on_day"
-          i18n_vars[:day] = I18n.l( search_date, format: :long )
-        end
-        if search_user
-          key += "_by_user"
-          i18n_vars[:user] = search_user.try_methods(:name, :login)
-        end
-        I18n.t( key, i18n_vars.merge( default: I18n.t( :observations_of_taxon, taxon_name: i18n_vars[:taxon] ) ) )
-      elsif search_user
-        if search_date
-          I18n.t( :observations_by_user_on_date, user: search_user.try_methods(:name, :login), date: I18n.l( search_date, format: :long ) )
-        else
-          I18n.t( :observations_by_user, user: search_user.try_methods(:name, :login) )
-        end
-      else
-        I18n.t( :x_site_is_a_social_network_for_naturalist, site: @site.name )
-      end
+      @shareable_description = begin
+                                generate_shareable_description
+                               rescue StandardError
+                                ""
+                               end
```

Comment:
Could you a) fix the indentation here, and b) put `Logstasher.write_exception( e, request: request, session: session, user: current_user )` in the `rescue` block so if we *are* having a lot of problems with generating the shareable description it will at least get logged?

Reply:
Sure, coming right up. Also, my fork is purely for contributing, so if there's something you'd like to get a quick change to, go right ahead! Doesn't bother me at all, but thanks for checking 😄 

One thing I noticed on some of the older files is that there is trailing whitespace on many lines so I turned off the auto correct indentation/whitespace for the project, so sorry I missed this one. 

In the future, if I touch a file with say one line change and a hundred minor whitespace changes, is that a problem?

Reply:
Not a major problem but also not the best practice for record-keeping / git blaming. I do similar stuff when I try to correct JS for eslint compliance, but I try to keep those to single commits so it's clear I wasn't trying to change any functionality. In the ruby code, I generally try to leave things alone unless I'm editing that line anyway.